packages/uswds-core/src/styles/mixins/helpers/checkbox-and-radio-colors.scss

```diff
@@ -124,6 +137,41 @@
 
   @if $input-type == "checkbox" {
     .usa-checkbox__input {
+      // Unchecked radio buttons receive :indeterminate pseudo-class.
+      // Because of this, we need to explicetly target checkbox :indeterminate styles.
```

Comment:
I thought this note would add clarity as to why we aren't including the styles below with lines 127-132 (which is where they were previously defined).

I wanted to document the decision here so that if future devs know to not move them in case they miss the `radio:indeterminate` regressions as we did in early testing.

If we don't think it's necessary, we can remove!

Reply:
It'd be helpful to speak to that in the comment.

```scss
// Adding indeterminate styles here avoids style conflicts from setting in lines 127-129.
```

---

>I wanted to document the decision here so that if future devs know to not move them in case they miss the radio:indeterminate regressions as we did in early testing.

Does that mean moving the styles to `127-132` will cause similar regressions with checkbox? If so, we can just change `radio → checkbox` because it looks like one affects the other.